Documentation/README.md

```diff
@@ -1,5 +1,7 @@
 # Rook
 
+**The Rook Cassandra operator is deprecated**
```

Comment:
worth linking these docs messages to the main README message, that has a bit more context/content, i.e. what operator we are recommending going forward?

Reply:
Links added, just note they won't work until after it's merged